### DISCUSSION TITLE
Make notarize support app_store_connect_api_key

### DISCUSSION THREAD

**Discussion by sarunw** [fastlane/fastlane#19292] [Ideas]
Most actions support [`app_store_connect_api_key`](https://docs.fastlane.tools/app-store-connect-api/), but right now [`notarize`](https://docs.fastlane.tools/actions/notarize/) action support only JSON way via `api_key_path` params.

It would be great if this action can support `app_store_connect_api_key` to work seamlessly along with other actions.

**Comment by macblazer** [comment]
This was done in PR #19353 which was merged on Sept 15, 2021.  It was released in [fastlane v2.194.0](https://github.com/fastlane/fastlane/releases/tag/2.194.0)
